packages/overlays/src/OverlayController.js

```diff
@@ -103,6 +103,18 @@ export class OverlayController {
    * @param {OverlayConfig} cfgToAdd
    */
   updateConfig(cfgToAdd) {
+    // only updating the viewportConfig
+    if (Object.keys(cfgToAdd).length === 1 && Object.keys(cfgToAdd)[0] === 'viewportConfig') {
```

Comment:
What would be use cases for these? (only providing `viewportConfig ` or only `popperConfig`)?
As of now, the only valid case would be to both specify this and `placementMode`, right?

Reply:
Yeah to be fair, this doesn't work as I want because as soon as another key in the configuration gets passed, it stops working. We simply need to fix this at the root cause (the init part that makes it close). I'll remove these ifs tomorrow.